File: Decorator/Java/CoffeeShopNoDecorator/src/CoffeeShopNoDecorator/DarkRoastWithMilk.java
package CoffeeShopNoDecorator;

public class DarkRoastWithMilk extends Beverage
{
    public DarkRoastWithMilk()
    {
        description = "Dark roast with milk";
    }

	@Override
    public double getPrice()
    {
        return 1.1d + 0.2d;
    }


}

File: Decorator/C#/CoffeeShopNoDecorator/DarkRoastWithMilk.cs
namespace CoffeeShopNoDecorator
{
    public class DarkRoastWithMilk : Beverage
    {
        public DarkRoastWithMilk()
        {
            Description = "Dark roast with milk";
        }

        public override decimal GetPrice()
        {
            return 1.1m + 0.2m;
        }

    }
}